pkgs/stream_channel/pubspec.yaml

```
@@ -1,5 +1,5 @@
 name: stream_channel
-version: 2.1.3
+version: 2.1.4
```

```suggestion
version: 2.2.0
```

This is technically a new API surface area. It _looks_ like a bug fix because of the naming, but in practice this is adding a feature. We should expect folks using this class as a mixin to use `^2.2.0` as a constraint.

---

I don't consider it adding a feature, just restoring an old feature that was accidentally removed - so a bug fix?

---

(specifically, 2.1.3 accidentally removed this feature which was breaking)